src/integrations/bluetooth-classic/bluetooth-classic.service.ts

```diff
@@ -31,6 +32,14 @@ import { DeviceTracker } from '../../entities/device-tracker';
 import { RoomPresenceProxyHandler } from '../room-presence/room-presence.proxy';
 import { BluetoothService } from '../bluetooth/bluetooth.service';
 
+interface DeviceInfo {
```

Comment:
I would rather use the already configured `Device` type from the home-assistant folder (device.ts). Allows us to ensure type safety across integrations, if Home Assistant ever changes its input format. Should just be a search and replace change, the structure of the object is the same.

Reply:
Unfortunately, there is already a BT Classic Device class (i.e. `src/integrations/bluetooth-classic/device.ts`) that appears similar but different to the Home Assistant definition (i.e. `src/integrations/home-assistant/device.ts`). I can dig a bit deeper and look to see if the two classes can be rationalised or alternatively rename the BT Classic class to avoid conflict ?

Reply:
Oh yeah, I missed that. I think you can just alias the import to avoid the name conflict (`import { Device as DeviceInfo } from '../home-assistant/device';`), but I'd also be fine with renaming the `bluetooth-classic/device.ts` to e.g. BluetoothDevice.

Reply:
I like the alias approach as I'm not really creating a new `Device` but rather defining an interface for the Device Info attributes. Managed to dig out an old BT Classic device and confirmed the right attributes come through.